arangod/V8Server/v8-actions.cpp

```diff
@@ -436,6 +436,62 @@ v8::Handle<v8::Object> TRI_RequestCppToV8(v8::Isolate* isolate,
   // intentional copy, as we will modify the headers later
   auto headers = request->headers();
 
+  switch (request->acceptEncoding()) {
+  case EncodingType::UNSET:
+    headers.insert(std::pair<std::string, std::string>(StaticStrings::AcceptEncoding, StaticStrings::EncodingIdentity));
```

Comment:
This change will mean that Foxx will now see a lot of requests coming in with `Accept-Encoding: identity`, however, this is not necessary what the client has actually sent.
Why is this change necessary?

Reply:
we could also store the original string, since there is more than `identity` and `deflate` in the world anyways. Then this can be the proper string.

Reply:
accept encoding wasn't filtered out as the other headers were - hence removed again.